_includes/core/linux_gs.html

```diff
@@ -10,7 +10,7 @@
           <p><span class="prompt unix-prompt"></span>sudo apt-get install libunwind8 libssl-dev unzip </p>
         </div>
       </div>
-      <p>Secondly, we need to install Mono (a temporary solution for now).</p>
+      <p>Secondly, we need to install <a href="http://www.mono-project.com">Mono</a> to be able to work with NuGet packages in later steps (a temporary solution for now).</p>
```

Comment:
We can probably be more specific than "work with NuGet packages". It's needed to support the NuGet client, which isn't yet supported by .NET Core.